wiki/platform_details.md

```diff
@@ -42,6 +42,9 @@ Each Learning Module will be no longer than 10 pages of content ( with pictures/
 - Link to Leadership board
 
 ### 3. Profile settings
+- Select Location (locate compost centers and upcycle organizations based on location) 
+- Resources for proper containment of waste (attain a metal trash bin for a home unit with pest issues/proper home sorting bins)
```

Comment:
Both of these first two sound great, but how would the location-based information work? Is there an already existing API for composting centers that we can consume? Otherwise it would be a massive amount of work for our first iteration of this app.